packages/blocks/src/store/types.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * Internal dependencies
+ */
+import type {
+	BlockCategory,
+	BlockCollection,
+	BlockVariation,
+	BlockStyle,
+	BlockType,
+} from '../types';
+
```

Comment:
@TimothyBJacobs would you recommend explicitly typing out the action types here and using the union of `Action` as the return type of all the action creators?